testing/softhsm/APKBUILD

```diff
@@ -0,0 +1,42 @@
+# Contributor: Frank Felhoffer <silveraid@hackme.ca>
+# Maintainer: Frank Felhoffer <silveraid@hackme.ca>
+pkgname=softhsm
+pkgver=2.3.0
+pkgrel=0
+pkgdesc="cryptographic store accessible through a PKCS #11"
+url="http://www.softhsm.org/"
+arch="all"
+license="BSD2"
+depends="libressl"
+makedepends="libressl-dev automake autoconf libtool file"
+install=""
+subpackages="$pkgname-doc"
+source="softhsm-$pkgver.tar.gz::https://github.com/opendnssec/SoftHSMv2/archive/$pkgver.tar.gz
+        01_aes_mac_fix.patch"
+builddir="$srcdir/SoftHSMv2-2.3.0"
+patch_args="-p1"
+
+prepare() {
+        default_prepare
+        cd "$builddir"
+        sh autogen.sh
+}
+
+build() {
+        cd "$builddir"
+        ./configure \
+                --prefix=/usr \
+                --sysconfdir=/etc \
+                --localstatedir=/var \
+                --disable-gost
```

Comment:
Could you add `--disable-static` too? Otherwise the large and useless `.a` gets bundled in.

Reply:
Added, thanks for the suggestion.